codespell_lib/data/dictionary.txt

```diff
@@ -7522,6 +7540,7 @@ famoust->famous
 fanatism->fanaticism
 fancyness->fanciness
 Farenheit->Fahrenheit
+farest->farthest, fairest,
```

Comment:
I think fairest should be the first suggestion @Gelma .

Reply:
Done